docroot/modules/custom/nvli_custom_search/src/Plugin/Block/CustomSearchBlock.php

```diff
@@ -32,7 +32,11 @@ protected function blockAccess(AccountInterface $account) {
    * {@inheritdoc}
    */
   public function build($keyword = NULL) {
+    $build = [];
     $keyword = \Drupal::request()->get('keyword');
-    return \Drupal::formBuilder()->getForm('Drupal\nvli_custom_search\Form\CustomSearchForm', $keyword);
+    $build['#cache']['max-age'] = 0;
```

Comment:
@dishabhadra Why we setting the max-age for it to be 0? I think the form block will always be the same. Would be good to cache i think.